auth/routes.js

```diff
@@ -28,30 +29,28 @@ router.post('/signin', auth, (req, res, next) => {
   // response sends back token, user, 
   res.cookie('auth', req.token);
   res.send(req.token);
-
-  // lab requires different shape response 
-  // res.send( {
-    // token: req.token,
-    // user: req.user,
-    // })
-  });
+});
 
 router.get();
```

Comment:
This is causing an error, because when you use a .get it won't work unless you have a path parameter!